pkg/controller/v1alpha2/experiment/managerclient/managerclient.go

```diff
@@ -84,7 +84,7 @@ func (d *DefaultClient) ValidateAlgorithmSettings(inst *experimentsv1alpha2.Expe
 
 func getExperimentConf(instance *experimentsv1alpha2.Experiment) *api_pb.Experiment {
 	experiment := &api_pb.Experiment{
-		Spec:   getExperimentSpec(instance),
+		Spec: getExperimentSpec(instance),
```

Comment:
Why do we need the change?

Reply:
Just run `make check`, you will got error:

```
go generate ./pkg/... ./cmd/...
hack/verify-gofmt.sh
diff -u ./pkg/controller/v1alpha2/experiment/managerclient/managerclient.go.orig ./pkg/controller/v1alpha2/experiment/managerclient/managerclient.go
--- ./pkg/controller/v1alpha2/experiment/managerclient/managerclient.go.orig	2019-05-28 19:00:34.080787976 -0700
+++ ./pkg/controller/v1alpha2/experiment/managerclient/managerclient.go	2019-05-28 19:00:34.080787976 -0700
@@ -84,7 +84,7 @@

 func getExperimentConf(instance *experimentsv1alpha2.Experiment) *api_pb.Experiment {
 	experiment := &api_pb.Experiment{
-		Spec:   getExperimentSpec(instance),
+		Spec: getExperimentSpec(instance),
 		Status: &api_pb.ExperimentStatus{
 			StartTime:      commonv1alpha2.ConvertTime2RFC3339(instance.Status.StartTime),
 			CompletionTime: commonv1alpha2.ConvertTime2RFC3339(instance.Status.CompletionTime),

Please run hack/update-gofmt.sh
```

That's why I want to add `make check` to prow.

Reply:
Gotcha